packages/component/src/Activity/CarouselFilmStrip.js

```diff
@@ -171,7 +171,6 @@ const CarouselFilmStrip = ({
 
   return (
     <div
-      aria-labelledby={ariaLabelId}
```

Comment:
Should we also remove all references to `ariaLabelId`? On line 130 and 203.

Reply:
Updated, thanks